lib/components/Box/useBoxStyles.treat.ts

```diff
@@ -370,3 +370,9 @@ export const relativePosition = {
     mapToStyleProperty(relativePositionRules, 'right'),
   ) as PositionRulesType,
 };
+
+export const userSelect = styleMap({
+  auto: { userSelect: 'auto' },
+  none: { userSelect: 'none' },
+  text: { userSelect: 'text' },
```

Comment:
Why is `text` required?

Reply:
As `auto` can select sub-element if not constrained, it seems given more granular options would be better

Reply:
Can you give me an example of where you've used this?

Reply:
The hesitation here is that these rules are sent to _every_ user regardless of whether they are used or not, so we tend to only add commonly used values.

Reply:
> Can you give me an example of where you've used this?

For instance, DB rebranded homepage has two accordions. Double clicked on its toggle might select all items in the target container occasionally.